src/main/resources/com/cloudbees/hudson/plugins/folder/AbstractFolder/tasks-top.jelly

```diff
@@ -25,7 +25,8 @@ THE SOFTWARE.
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout"
          xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:i="jelly:fmt" xmlns:this="this">
   <j:set var="url" value="${h.getNearestAncestorUrl(request,it)}"/>
-  <l:task icon="icon-search icon-md"  href="${url}/" title="${%Status}" contextMenu="false" />
+  <l:task icon="icon-up icon-md" href="${url}/../../" title="${%Up}" contextMenu="false"/>
```

Comment:
I thought we were getting rid of these? Or is that a separate PR?

Reply:
separate, https://github.com/jenkinsci/cloudbees-folder-plugin/pull/252 I suspect a bad merge resolution